quartz.config.ts

```diff
@@ -9,6 +9,7 @@ const config: QuartzConfig = {
     analytics: {
       provider: "plausible",
     },
+    locale: "fr-FR",
```

Comment:
```suggestion
    locale: "en-US",
```

let's set this by default.

Reply:
Oh, yes, sorry, forgot to remove that!